src/main/scala/scorex/core/transaction/wallet/BoxWallet.scala

```diff
@@ -11,31 +11,21 @@ import scorex.core.{ModifierId, NodeViewModifier, PersistentNodeViewModifier}
 
 import scala.util.Try
 
-//TODO why do we need transactionId and createdAt
-case class WalletBox[P <: Proposition, B <: Box[P]](box: B, transactionId: Array[Byte], createdAt: Long)
-                                                   (subclassDeser: Serializer[B]) extends BytesSerializable
+case class WalletBox[P <: Proposition, B <: Box[P]](box: B)(subclassDeser: Serializer[B]) extends BytesSerializable
```

Comment:
Because these are useless fields that does not provide any value.
`transactionId` is usually included inside a box (e.g. we have it in `ErgoBox`), `createdAt` is implicitly included in transactinId and does not provide any additional information.

Reply:
WalletBox is not used in Scorex. It is being used in Twinscoin where a box does not contain a link to a corresponding transaction. The fields being removed could be useful for developments of the Twinscoin protocol (e.g. adding coin-age conditions to the PoS inequality) and wallet (e.g. building transaction <-> box links, currently HBoxWallet is not implementing transaction storage at all, see historyTransactions field)

Reply:
But these fields are not used in twinscoin either.
Should I move the hole class to examples?

Reply:
WalletBox / BoxWallet as well as Blockchain etc classes concretizing rather abstract core functionalities were planned to be in a "mid" layer, which is not even started yet.